test/functional/outputs/forward_to_elasticsearch_test.go

```diff
@@ -0,0 +1,68 @@
+package outputs
+
+import (
+	"github.com/ViaQ/logerr/log"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/test/functional"
+	"github.com/openshift/cluster-logging-operator/test/helpers/types"
+	"github.com/openshift/cluster-logging-operator/test/matchers"
+	"github.com/openshift/cluster-logging-operator/test/runtime"
+)
+
+var _ = Describe("[Functional][Outputs][ElasticSearch] FluentdForward Output to ElasticSearch", func() {
+
+	const (
+		elasticSearchImage = "elasticsearch:7.10.1"
+	)
+
+	var (
+		framework *functional.FluentdFunctionalFramework
+
+		newVisitor = func(f *functional.FluentdFunctionalFramework) runtime.PodBuilderVisitor {
+			return func(b *runtime.PodBuilder) error {
+				log.V(2).Info("Adding ElasticSearch output container", "name", logging.OutputTypeElasticsearch)
+				b.AddContainer(logging.OutputTypeElasticsearch, elasticSearchImage).
+					AddEnvVar("discovery.type", "single-node").
+					AddRunAsUser(2000).
+					End()
+				return nil
+			}
+		}
+
+		// Template expected as output Log
+		outputLogTemplate = functional.NewApplicationLogTempate()
+	)
+
+	BeforeEach(func() {
+
+		framework = functional.NewFluentdFunctionalFramework()
+		addElasticSearchContainer := newVisitor(framework)
+		functional.NewClusterLogForwarderBuilder(framework.Forwarder).
+			FromInput(logging.InputNameApplication).
+			ToElasticSearchForwardOutput()
+		Expect(framework.DeployWithVisitor(addElasticSearchContainer)).To(BeNil())
+		Expect(framework.WritesApplicationLogs(1)).To(BeNil())
+	})
+	AfterEach(func() {
+		framework.Cleanup()
+	})
+
+	Context("when sending to ElasticSearch using ElasticSearch's forward protocol", func() {
```

Comment:
it's just sending to Elasticsearch, not specific to fluent's forward protocol.  Can we also mention the version being tested here since that is the specific goal of adding the test

Reply:
Done